includes/transformer/class-base.php

```diff
@@ -271,6 +271,10 @@ public function get_replies() {
 	 * @return array The content map for the post.
 	 */
 	protected function get_content_map() {
+		if ( ! \method_exists( $this, 'get_content' ) || ! $this->get_content() ) {
+			return null;
```

Comment:
Return type for this method is array, should we return an empty array here instead? Or update the docs?

Reply:
Update the docs. With an empty array, it will be added to the output, with `null`, it will be ignored.